script.sh

```diff
@@ -98,7 +98,17 @@ echo '::group:: Running tflint with reviewdog 🐶 ...'
   set +Eeuo pipefail
 
   # shellcheck disable=SC2086
-  TFLINT_PLUGIN_DIR=${TFLINT_PLUGIN_DIR} "${TFLINT_PATH}/tflint" -c "${INPUT_TFLINT_CONFIG}" --format=checkstyle ${INPUT_FLAGS} --chdir=${INPUT_TFLINT_TARGET_DIR} \
+
+  # We only want to specify the tflint target directory if it is not the default to avoid conflicts
+  export CHDIR_COMMAND=""
```

Comment:
```suggestion
  CHDIR_COMMAND=""
```
I believe `export` isn't needed here. This isn't an environment variable which you intend to make available to a child process of this script.